conf/messages

```diff
@@ -258,6 +263,31 @@ paye.ats.treasury_spending.para2=All figures are rounded to the nearest pound.
 paye.ats.treasury_spending.para3=The figures in the table above are intended as an illustration of how taxes are spent and not as a direct link between your Income Tax, National Insurance contributions and any specific expenditure.
 paye.ats.treasury_spending.para4=Spending information is published by HM Treasury.
 
+# View ATS - Summary (PAYE)
+#============================================================
+paye.ats.summary.html.title=Your income and taxes
+paye.ats.summary.title=Your income and taxes
```

Comment:
Same message

Reply:
removed the first one as we are not using it anywhere